src/panels/lovelace/editor/card-editor/hui-edit-card.ts

```diff
@@ -77,6 +81,83 @@ export class HuiEditCard extends hassLocalizeLitMixin(LitElement) {
     };
   }
 
+  static get styles(): CSSResult[] {
+    return [
+      css`
```

Comment:
Put haStyleDialog in the array before our own CSS and it should work ?

Reply:
Nope... it doesn't :-(

Reply:
Using `styles()` gives:
![image](https://user-images.githubusercontent.com/5662298/51342887-5335e100-1a95-11e9-9f97-7d49fd466215.png)

Using `${haStyleDialog.cssText}` gives:
![image](https://user-images.githubusercontent.com/5662298/51342933-73fe3680-1a95-11e9-8680-f78b42683b85.png)

Reply:
Okay, so let's rewrite this CSS without apply. Apply is just a bunch of rules. That we can copy in it's place instead. Even better, we could compose it using vars.

```ts
const haDialogNarrowStyle = css`
  margin: 0;
`;

const haDialogStlye = css`
  @media all and (max-width: 450px), all and (max-height: 500px) {
    paper-dialog {
      ${haDialogNarrowStyle}
    }
  }
`
```